packaging/opae/rpm/create

```diff
@@ -66,56 +83,30 @@ do
     mkdir -p "${RPMBUILD_DIR}/$d"
 done
 
-# Create source tarball.
+# clone opae legancy and tests
+
 cd "${SDK_DIR}/.."
-tar --transform='s/opae-sdk/opae/' \
-  --exclude=.git \
-  --exclude=.gitignore \
-  --exclude=.github \
-  --exclude=.travis.yml \
-  --exclude=opae.spec.in \
-  --exclude=opae.spec \
-  --exclude=opae-libs/external \
-  --exclude=opae-libs/tests \
-  --exclude=opae-libs/plugins/ase \
-  --exclude=opae-libs/cmake/config/libopae-all.spec.in \
-  --exclude=opae-libs/cmake/config/run_coverage_test.sh.in \
-  --exclude=opae-libs/cmake/config/run_coverage_test_local.sh.in\
-  --exclude=external \
-  --exclude=platforms \
-  --exclude=samples/base \
-  --exclude=samples/hello_afu \
-  --exclude=samples/hello_mpf_afu \
-  --exclude=samples/intg_xeon_nlb \
-  --exclude=samples/base \
-  --exclude=scripts \
-  --exclude=tests \
-  --exclude=tools/fpgametrics \
-  --exclude=tools/libboard/board_dc \
-  --exclude=tools/extra/ras \
-  --exclude=tools/extra/fpgabist \
-  --exclude=tools/extra/pac_hssi_config \
-  --exclude=tools/extra/fpgadiag \
-  --exclude=tools/extra/c++utils \
-  --exclude=tools/extra/pyfpgadiag \
-  --exclude=tools/extra/pypackager \
-  --exclude=tools/utilities \
-  --exclude=opae-libs/include/opae/cxx/.clang-format \
-  --exclude=opae-libs/libopaecxx/.clang-format \
-  --exclude=opae-libs/pyopae/.clang-format \
-  --exclude=opae-libs/.clang-format \
-  --exclude=opae-libs/.clang-format \
-  -z -c -f opae.tar.gz opae-sdk
-
-mv opae.tar.gz "${RPMBUILD_DIR}/SOURCES"
+rm -rf _build
+mkdir _build
+cd _build
+cmake .. -DOPAE_BUILD_LEGACY=ON  -DOPAE_BUILD_TESTS=ON
```

Comment:
This command is erroring out because line 88 changes directory to the parent of the codebase root directory (${SDK_DIR}).
A better way would be to not change directories and use source/build paths when calling cmake.

```bash
CMAKE_BUILD_DIR="${SDK_DIR}/_build"
rm -rf ${CMAKE_BUILD_DIR}
mkdir ${CMAKE_BUILD_DIR}
cmake -S ${SDK_DIR} -B ${CMAKE_BUILD_DIR
```

Reply:
fixed